tests/changelog.spec.js

```diff
@@ -0,0 +1,58 @@
+import { test, expect } from "@playwright/test";
+const AxeBuilder = require("@axe-core/playwright").default;
+
+// Changelog Page
+test("Changelog has title", async ({ page }) => {
+  await page.goto("/changelog");
+  await expect(page).toHaveTitle("LinkFree user changelog");
+});
+
+// Navigating to Changelog Page
+test("Navigate to Changelog", async ({ page }) => {
+  await page.goto("/roadmap");
+  await page.getByRole("link", { name: "See full list" }).click();
+  await expect(page.locator("h1")).toHaveText("Changelog");
+});
+
+// Changelog Listed
+test("Changelog listed", async ({ page }) => {
+  await page.goto("/changelog");
+  const listItems = await page.$$("main li"); // Get all the list items inside the main element
```

Comment:
Thank you, it is now passing but from the official docs this is not recommended

<img width="1660" alt="Screenshot 2023-06-05 at 13 01 04" src="https://github.com/EddieHubCommunity/LinkFree/assets/624760/2ac9161f-e422-4123-a3b4-d39eab7cfa69">

Reply:
okay so should I change the code and work on it?

Reply:
yes please update the test to use the `page.locator` as it recommends in the docs